.gitignore

```diff
@@ -27,3 +27,6 @@ npm-debug.*
 spec/dummy/client/npm-debug.log.*
 
 /gen-examples
+
+# Idea
+.idea/
```

Comment:
always have a empty line (no red in the PR)

FWIW, I consider the .idea directory something that you have to put in your global git ignore.